simulator/learned_model/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.14.4)
+project(learned_model)
```

Comment:
Please change the package name. Just `lerned_model` is not clear for other users. It'd be `ml_based_vehicle_model` or `learning_based_vehicle_model`?